SwiftCompilerSources/Sources/AST/Conformance.swift

```diff
@@ -60,6 +60,16 @@ public struct Conformance: CustomStringConvertible, NoReflectionChildren {
     return bridged.getInheritedConformance().conformance
   }
 
+  public var rootConformance: Conformance {
+    if isInherited {
+      return inheritedConformance.rootConformance
+    }
+    if isSpecialized {
+      return genericConformance
```

Comment:
Does `var genericConformance` need to exist at all then?

Reply:
I think so